### ISSUE TITLE
swine herder and lumberjack huts causing crash upon breaking

### ISSUE THREAD

**Issue by anthonyispenguin** [ldtteam/minecolonies#3846]
### Minecolonies version

Version: 1.14.4-0.10.358-ALPHA-universal

### Expected behavior

though textured in inventory they function as should, so far. 
- 
Played the game as normal, the blocks still serve their purpose on job sites
### Actual behaviour

Blocks missing textures when placed in the world, and crash the game if broken.
- 
Bit of a no brainer, but I'd figured I'd try and see if I could remove them no problem. As it turns out it became a problem.

![2019-09-13_22 52 07](https://user-images.githubusercontent.com/55305774/64903172-db4aa580-d679-11e9-920d-0759cac0d904.png)
![2019-09-13_22 52 16](https://user-images.githubusercontent.com/55305774/64903173-dd146900-d679-11e9-8d87-e025c2956c00.png)

### Steps to reproduce the problem

1. enter any world on creative
2. place town hall, doesn't matter if built or not.
3A. place lumber jack hut
3B.Place swineherder hut
4 enter survival
5 break either block
6A. Game crash
6B. Game crash
[crash-2019-09-13_22.52.42-client.txt](https://github.com/ldtteam/minecolonies/files/3612322/crash-2019-09-13_22.52.42-client.txt)
[crash-2019-09-13_22.58.53-client.txt](https://github.com/ldtteam/minecolonies/files/3612323/crash-2019-09-13_22.58.53-client.txt)

**Comment by SoNick** [comment]
Similar issue with the Citizen's Hut as well; the block is textured however attempting to break it causes the game to crash with the below message. Particles while breaking are the purple-and-black checkerboard image used for missing textures.

Error: java.lang.IllegalArgumentException: Cannot get property EnumProperty{name=type, clazz=class net.minecraft.state.properties.ChestType, values=[SINGLE, LEFT, RIGHT]} as it does not exist in Block{minecolonies:blockhutcitizen}

**Comment by anthonyispenguin** [comment]
> 
> 
> Similar issue with the Citizen's Hut as well; the block is textured however attempting to break it causes the game to crash with the below message. Particles while breaking are the purple-and-black checkerboard image used for missing textures.
> 
> Error: java.lang.IllegalArgumentException: Cannot get property EnumProperty{name=type, clazz=class net.minecraft.state.properties.ChestType, values=[SINGLE, LEFT, RIGHT]} as it does not exist in Block{minecolonies:blockhutcitizen}

 I'll also add there's a similar bug with the town hall as well. Just had it happen when I was starting a modded survival anded need to place it down somewhere else.

**Comment by bicobus** [comment]
All hut blocks cause crashes upon breaking.

**Comment by anthonyispenguin** [comment]
> 
> 
> All hut blocks cause crashes upon breaking.


I figured I'd try out every block just to be sure, and it is surely the case. I have found an alternative to breaking the blocks is to reposition them. However, if the hut is already built, it's going to require tearing down the building manually. It's only real trouble when the hut is many blocks away from it's initial position, that unfortunately require precision click spamming.
